src/database/schema/UserSchema.ts

```diff
@@ -335,7 +347,10 @@
         anon_user_info_color_code: null,
         anon_user_info_color_name: null,
         anon_user_info_emoji_code: null,
-        anon_user_info_emoji_name: null
+        anon_user_info_emoji_name: null,
+        isAnonymous: IS_NOT_ANONYMOUS
+
+        // TODO: add is_anonymous
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, please remove the TODO comment if the task has been completed.

```diff
-        // TODO: add is_anonymous
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->